python/api/onnxruntime-python-api.py

```diff
@@ -0,0 +1,117 @@
+# A set of code samples showing different usage of the ONNX Runtime Python API
```

Comment:
Can also refer to C:\Users\prs\work_projects\onnxruntime\onnxruntime\test\python\onnxruntime_test_python_iobinding.py and 
C:\Users\prs\work_projects\onnxruntime\onnxruntime\test\python\onnxruntime_test_python.py

Reply:
I'm hoping these can be self contained samples